devDocs/githubIssueTemplateExplanationAndExamples.md

```diff
@@ -178,14 +178,14 @@ It can help us determine if changes in NVDA or other software introduced the pro
 > * NVDA 2018.1 - no error sound
 > * NVDA 2018.2 - has error sound
 
-#### If addons are disabled, is your problem still occurring?
+#### If NVDA add-ons are disabled, is your problem still occurring?

```

Comment:
I think we are trying to use "addons" over "add-ons"

```suggestion
#### If NVDA addons are disabled, is your problem still occurring?
```

Reply:
Do we? User guide and NVDA's interface still use form with a hyphen. Any particular reason for changing that?

Reply:
> I think we are trying to use "addons" over "add-ons"

Are you sure? Can you confirm it with other devs?
On the web, "add-on" with hyphen seems more current than "addon" without it. Cf. Wikipedia, Mozilla, etc.

A few month ago I was told to replace hyphenless "addon" with hyphenated "add-on" in my add-on's documentation. Of course, it's not NVDA's core. But if a decision is made on this point, it would be interesting to communicate it.

Note that "addon" without hyphen is widely used in any case in more technical syntagms such as file names, variable or function names, URLs, name of mailing list, etc.

Reply:
I was mistaken, going off what we've been using for the "addon store".

Reply:
I have pushed 2e7ccf6e922f947547bcf7e3680a72773140fbbf to uniformize the writing of "add-on" with the dash on the files of this PR. If you confirm that the dash should be used, think to uniform it when working on other topics, especially on add-on store.